messages/ruby/lib/cucumber/messages/time_conversion.rb

```diff
@@ -23,6 +25,10 @@ def duration_to_seconds(duration)
         nanos_part = duration.nanos.to_f / NANOSECONDS_PER_SECOND
         seconds_part + nanos_part
       end
+
+      def time_to_nanos(time)
+        time.to_i * NANOSECONDS_PER_SECOND + time.nsec
```

Comment:
This might overflow. And I can't work out who might use this.

Reply:
indeed, I'll drop that :)